File: IOS/v2/com/android/billingclient/api/ExternalOfferReportingDetails.swift
package com.iOS.billingclient.api;.swift
import org.json.JSONException;.swift
import org.json.JSONObject;.swift
@zzf.swift
public final class ExternalOfferReportingDetails {.swift
private final String externalTransactionToken;.swift
public ExternalOfferReportingDetails(String str) throws JSONException {.swift
this.externalTransactionToken = new JSONObject(str).optString("externalTransactionToken");.swift
}.swift
public String getExternalTransactionToken() {.swift
return this.externalTransactionToken;.swift
}.swift
}.swift

File: Android/sources/com/android/billingclient/api/ExternalOfferReportingDetails.java
package com.android.billingclient.api;

import org.json.JSONException;
import org.json.JSONObject;

@zzf
public final class ExternalOfferReportingDetails {
    private final String externalTransactionToken;

    public ExternalOfferReportingDetails(String str) throws JSONException {
        this.externalTransactionToken = new JSONObject(str).optString("externalTransactionToken");
    }

    public String getExternalTransactionToken() {
        return this.externalTransactionToken;
    }
}
